src/qt/buynamespage.cpp

```diff
@@ -45,22 +49,61 @@ bool BuyNamesPage::eventFilter(QObject *object, QEvent *event)
 {
     if (event->type() == QEvent::FocusIn)
     {
-        if (object == ui->registerName)
+        if (object == ui->registerNameAscii)
+        {
+            ui->registerNameButton->setDefault(true);
+        }
+
+        if (object == ui->registerNameHex)
+        {
+            ui->registerNameButton->setDefault(true);
+        }
+
+        if (object == ui->registerNameDomain)
         {
             ui->registerNameButton->setDefault(true);
         }
+
     }
     return QWidget::eventFilter(object, event);
 }
 
-void BuyNamesPage::onNameEdited(const QString &name)
-{
-    if (!walletModel)
-        return;
 
-    const QString availableError = name_available(name);
+QString BuyNamesPage::DomainToASCII(const QString &name){
+    
```

Comment:
I personally think this blank line can be removed, but if it's not removed, at least remove the spaces from this line.